app/models/google_event.rb

```diff
@@ -55,6 +57,10 @@ def create(credentials, conference_room_id, raw_event_data = {})
     end
 
     def insert_event_and_return_result(credentials, event_data)
+      if any_events_in_timespan?(credentials, event_data[:attendees].first,
+                                 event_data[:start][:date_time], event_data[:end][:date_time])
+        raise EventInTimeSpanError, 'Invalid time span'
```

Comment:
What do you think about adding more descriptive error message? Ie. what events are in that time span?